plugins/generator-1.14.4/forge-1.14.4/templates/modbase/mod.java.ftl

```diff
@@ -48,9 +48,12 @@
 
 package ${package};
 
+import org.apache.logging.log4j.Logger
+
 @Mod("${modid}")
 public class ${JavaModName} {
 
+    public static final Logger LOGGER = org.apache.logging.log4j.LogManager.getLogger("${JavaModName}.class");

```

Comment:
remove the `"` from `${JavaModName}.class`

Reply:
> remove the `"` from `${JavaModName}.class`

I think `"`s are needed

Reply:
I tested and it works with the "

Reply:
It isn't a matter of whether it works or not.
If you keep the `"` then the marker will be suffixed with a `.class` which looks ugly. 
`${JavaModName}.class` is enough

Reply:
`LogManager.getLogger` has an overload for `Class<?>`

Reply:
Yep indeed this should not use "", just reference Class<?> directly

Reply:
fixed!